docs/java-agent-injector.md

```diff
@@ -0,0 +1,168 @@
+# Java agent injector Manual
+
+In order to use java agent more natively , we propose the java agent injector to inject the agent sidecar to a pod.
+
+When enabled in a pod's namespace , the injector injects the java agent container at pod creation time using a mutating webhook admission controller. By rendering java agent to a shared volume , containers within the pod can use the java agent.
+ 
+The following sections describe how to configure agent, if you want to try it directly, please see [Usage](examples/java-agent-injector-usage) for more details.
+
+## Install Injector
+
+The java agent injector is builded in the operator , so you need to follow [Operator installation instrument](../README.md#operator) to install the operator firstly.
+
+## Active java agent injectoin
+
+We have two granularities here: namespace and pod.
+
+| Resource  | Label               | Enabled value | Disabled value |
+| --------- | ------------------- | ------------- | -------------- |
+| Namespace | swck-injection      | enabled       | disabled       |
+| Pod       | swck-java-agent-injected | "true"        | "false"        |
+
+The injector is configured with the following logic:
+
+1. If either label is disabled, the pod is not injected.
+2. If two labels are enabled, the pod is injected.
```

Comment:
IIRC, either label is enabled, the pod CAN be injected.

The pod granularity only helps users to enable injection for some specific pods which belong to a namespace without the `sw-injection` label.

Reply:
Let's keep it as is.